packages/okta-adapter/test/filters/expression_language.test.ts

```diff
@@ -0,0 +1,187 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import _ from 'lodash'
+import { ElemID, InstanceElement, ObjectType, ReferenceExpression, TemplateExpression, isInstanceElement } from '@salto-io/adapter-api'
+import { filterUtils } from '@salto-io/adapter-components'
+import { getFilterParams } from '../utils'
+import oktaExpressionLanguageFilter from '../../src/filters/expression_language'
+import { GROUP_RULE_TYPE_NAME, GROUP_TYPE_NAME, OKTA, POLICY_RULE_TYPE_NAME, USER_SCHEMA_TYPE_NAME } from '../../src/constants'
+
+describe('expression language filter', () => {
+      type FilterType = filterUtils.FilterWith<'onFetch' | 'onDeploy' | 'preDeploy'>
```

Comment:
you don't need this for now right? 
```suggestion
      type FilterType = filterUtils.FilterWith<'onFetch'>
```